tap-snapshots/test-parse-test.js-TAP.test.js

```diff
@@ -5189,6 +5558,133 @@ Node {
 }
 `
 
+exports[`test/parse-test.js TAP parse simple-await-template.js > must match snapshot 1`] = `
+Node {
+  "async": true,
+  "body": Node {
+    "end": 18,
+    "expressions": Array [
+      Node {
+        "argument": Node {
+          "end": 16,
+          "name": "promise",
+          "start": 9,
+          "type": "Identifier",
+        },
+        "end": 16,
+        "start": 3,
+        "type": "AwaitExpression",
+      },
+    ],
+    "quasis": Array [
+      Node {
+        "end": 1,
+        "start": 1,
+        "tail": false,
+        "type": "TemplateElement",
+        "value": Object {
+          "cooked": "",
+          "raw": "",
+        },
+      },
+      Node {
+        "end": 17,
+        "start": 17,
+        "tail": true,
+        "type": "TemplateElement",
+        "value": Object {
+          "cooked": "",
+          "raw": "",
+        },
+      },
+    ],
+    "start": 0,
+    "type": "TemplateLiteral",
+  },
+  "databaseClients": Map {},
+  "end": 19,
+  "fileAttachments": Map {},
+  "generator": false,
+  "id": null,
+  "references": Array [
+    Node {
+      "end": 16,
+      "name": "promise",
+      "start": 9,
+      "type": "Identifier",
+    },
+  ],
+  "secrets": Map {},
+  "start": 0,
+  "type": "Cell",
+}
+`
+
+exports[`test/parse-test.js TAP parse simple-await-template.md > must match snapshot 1`] = `
+Node {
+  "async": true,
+  "body": Node {
+    "end": 16,
+    "expressions": Array [
+      Node {
+        "argument": Node {
+          "end": 15,
+          "name": "promise",
+          "start": 8,
+          "type": "Identifier",
+        },
+        "end": 15,
+        "start": 2,
+        "type": "AwaitExpression",
+      },
+    ],
+    "quasis": Array [
+      Node {
+        "end": 0,
+        "start": 0,
+        "tail": false,
+        "type": "TemplateElement",
+        "value": Object {
+          "cooked": "",
+          "raw": "",
+        },
+      },
+      Node {
+        "end": 16,
+        "start": 16,
+        "tail": true,
+        "type": "TemplateElement",
+        "value": Object {
+          "cooked": undefined,
```

Comment:
Wanted to call out one small difference between the TemplateCell version and the JavaScript version. In the template version, the `cooked` value of the second quasi is `undefined` but in the JavaScript version, the same `cooked` value is `""`. I'm looking into the root cause for this now!

Reply:
(Compare line 5658 with 5596)